platform-integrations/bob/evolve-lite/skills/evolve-lite-recall/SKILL.md

```
@@ -80,7 +80,14 @@ Entities can come from multiple sources:
         alice-guideline.md                        <- annotated [from: alice]
 ```
 
-Each file uses markdown with YAML frontmatter:
+The manifest output is human-readable:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)